### PR TITLE
Add new changelog for 0.2.6-M1

### PR DESCRIPTION
**Pull request by tunacicek**

### CODE REVIEW

File: charts/semantic-hub/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ###############################################################
-# Copyright (c) 2021-2022 Robert Bosch Manufacturing Solutions GmbH
-# Copyright (c) 2021-2022 Contributors to the Eclipse Foundation
+# Copyright (c) 2021-2023 Robert Bosch Manufacturing Solutions GmbH
+# Copyright (c) 2021-2023 Contributors to the Eclipse Foundation
 #
```
Comment (carslen) [inline charts/semantic-hub/Chart.yaml:4]: Please use `2021, 2023` instead (also in the other files)

File: charts/LICENSE [hunk 2]
```diff
@@ -0,0 +1,202 @@
+
```
Comment (carslen) [inline charts/LICENSE:1]: This file is expected to be inside a chart folder, please move this file to `semantic-hub` folder → https://eclipse-tractusx.github.io/docs/release/trg-5/trg-5-02#chart-file-structure

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 0.2.6-M1
+### Added
+
+### Fixed
+- Update Dependencies.md
```
Comment (carslen) [inline CHANGELOG.md:11]: Will there be another PR with the updated `DEPENDENCIES` file? BTW, the file is named without _.md_.

**Review by carslen — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (tunacicek) [reply CHANGELOG.md:11]: Yes, there will be an other PR for updating the dependencies.

File: charts/LICENSE (hunk 2 above)
Comment (tunacicek) [reply charts/LICENSE]: done :)

File: charts/semantic-hub/Chart.yaml (hunk 1 above)
Comment (tunacicek) [reply charts/semantic-hub/Chart.yaml:4]: Done

**Review by carslen — CHANGES_REQUESTED** [review]
Helm testing fails: [Reason](https://github.com/eclipse-tractusx/sldt-semantic-hub/actions/runs/4862254465/jobs/8668369466?pr=109#step:10:136)

You did not build the required image yet.

Please consider to add `charts/semantic-hub/README.md` to this PR as mentioned in #108 . You can use [helm-docs](https://github.com/norwoodj/helm-docs#helm-docs) to create this file and maybe it's worth tweeking the `values.yaml` to get better results (by adding comments).
